profiles/schema.py

```diff
@@ -366,6 +373,39 @@ def mutate(self, info, **kwargs):
         return UpdateProfile(profile=profile)
 
 
+class DeleteProfile(graphene.Mutation):
+    ok = graphene.Boolean()
+
+    class Arguments:
+        id = graphene.ID(required=True)
```

Comment:
Is there a reason why this delete needs ID argument but the update does not? To me it seems a bit inconsistent.

Reply:
IMO using id instead of relying on user is better way to accomplish this and we have discussed and kinda agreed this before with people working on the project. Using id makes the same API usable for admin users and in next PR there will be actually use case for profiles that have no user.

Update should be refactored at some point but there's been too much of that lately so it's going to happen another time. Ticket filed.